### DISCUSSION TITLE
Chaos;Head Noah and Chaos;Child (Double Pack)

### DISCUSSION THREAD

**Discussion by pitakph** [skyline-emu/skyline#2210] [Q&A]
Skyline only detects Chaos;Head. Can I make it show both games?

Also, black screen with stuttering audio during cutscenes. I've noticed this with other games too. Can this be fixed?

**Comment by guranu** [comment]
@pitakph chaos head and chaos child are seperate games so if you want to play both of them dump chaos head and chaos child also the cutscene will be fixed when the devs add nvdec to the emulator.
